BucketeerTests/BKTConfigTests.swift

```diff
@@ -0,0 +1,100 @@
+import XCTest
+@testable import Bucketeer
+
+final class BKTConfigTests: XCTestCase {
+
+    func testCreateConfig() {
```

Comment:
Can you also add the following cases to check the exceptions?
https://github.com/bucketeer-io/ios-client-sdk/pull/13/files#diff-a01b127e0bdeebe208c5ce7cac0dfd53041a03921bc362c81bdb3954f333d81eR134-R145

Reply:
sure, I will add it. 
In the current test, if the user missing something, the BKTConfig will be `nil`

Reply:
added. Please continue review @cre8ivejp